src/DiadocApi.cs

```diff
@@ -330,34 +330,45 @@ public GeneratedFile GenerateInvoiceDocumentReceiptXml(string authToken, string
 			return diadocHttpApi.GenerateInvoiceDocumentReceiptXml(authToken, boxId, messageId, attachmentId, signer);
 		}
 
-		public GeneratedFile GenerateInvoiceCorrectionRequestXml(string authToken, string boxId, string messageId,
-			string attachmentId, InvoiceCorrectionRequestInfo correctionInfo)
+		public GeneratedFile GenerateInvoiceCorrectionRequestXml(string authToken,
+			string boxId,
+			string messageId,
+			string attachmentId,
+			InvoiceCorrectionRequestInfo correctionInfo)
 		{
 			if (boxId == null) throw new ArgumentNullException("boxId");
 			if (messageId == null) throw new ArgumentNullException("messageId");
 			if (attachmentId == null) throw new ArgumentNullException("attachmentId");
 			return diadocHttpApi.GenerateInvoiceCorrectionRequestXml(authToken, boxId, messageId, attachmentId, correctionInfo);
 		}
 
-		public GeneratedFile GenerateRevocationRequestXml(string authToken, string boxId, string messageId,
-			string attachmentId, RevocationRequestInfo revocationRequestInfo)
+		public GeneratedFile GenerateRevocationRequestXml(string authToken,
+			string boxId,
+			string messageId,
+			string attachmentId,
+			RevocationRequestInfo revocationRequestInfo)
 		{
 			if (boxId == null) throw new ArgumentNullException("boxId");
 			if (messageId == null) throw new ArgumentNullException("messageId");
 			if (attachmentId == null) throw new ArgumentNullException("attachmentId");
 			return diadocHttpApi.GenerateRevocationRequestXml(authToken, boxId, messageId, attachmentId, revocationRequestInfo);
 		}
 
-		public GeneratedFile GenerateSignatureRejectionXml(string authToken, string boxId, string messageId,
-			string attachmentId, SignatureRejectionInfo signatureRejectionInfo)
+		public GeneratedFile GenerateSignatureRejectionXml(string authToken,
+			string boxId,
+			string messageId,
+			string attachmentId,
+			SignatureRejectionInfo signatureRejectionInfo)
 		{
 			if (boxId == null) throw new ArgumentNullException("boxId");
 			if (messageId == null) throw new ArgumentNullException("messageId");
 			if (attachmentId == null) throw new ArgumentNullException("attachmentId");
 			return diadocHttpApi.GenerateSignatureRejectionXml(authToken, boxId, messageId, attachmentId, signatureRejectionInfo);
 		}
 
-		public InvoiceCorrectionRequestInfo GetInvoiceCorrectionRequestInfo(string authToken, string boxId, string messageId,
+		public InvoiceCorrectionRequestInfo GetInvoiceCorrectionRequestInfo(string authToken,
```

Comment:
Please, move first argument on the next line too.

Reply:
Decided to revert these changes as well.